package.json

```diff
@@ -66,19 +66,20 @@
     "cz-conventional-changelog": "2.1.0",
     "husky": "^0.14.3",
     "jest": "^22.0.4",
+    "jest-cli": "^22.4.3",
```

Comment:
installing jest does / should not require separate install to jest-cli. (it's dependency of `jest` itself). Please revert it back.